osquery/tables/networking/linux/iptables.cpp

```diff
@@ -35,6 +35,12 @@ static const std::string kHexMap = "0123456789ABCDEF";
 static const int kMaskHighBits = 4;
 static const int kMaskLowBits = 15;
 
+namespace {
+  const char* formatInvFlag(const iptcproxy_rule& rule, int flag) {
```

Comment:
I know it's technically less performant but I recommend returning a `std::string` here.